src/commands/lint/lint.constants.ts

```diff
@@ -0,0 +1,24 @@
+export const regions = [
+  'northland',
+  'auckland',
+  'waikato',
+  'bay-of-plenty',
+  'gisborne',
+  'hawkes-bay',
+  'taranaki',
+  'manawatu-whanganui',
+  'wellington',
+  'tasman',
+  'nelson',
+  'marlborough',
+  'west-coast',
+  'canterbury',
+  'otago',
+  'southland',
+];
+
+export const imageryBucketNames = ['linz-imagery', 'nz-imagery'];
+
+export const imageryProducts = ['rgb'];
+
+export const imageryCrs = ['2193'];
```

Comment:
Same as https://github.com/linz/argo-tasks/pull/727#discussion_r1379494614, we should probably use `EpsgCode.NZTM2000` here. Are there constants for the other values as well?

Reply:
Thanks for sharing that, I hadn't seen it.
I think only the EpsgCode exists at the moment? but it would be cool to see more added.

Reply:
Yeah, we should probably add any codes which are currently in use anywhere, and import them where necessary.